padertorch/io.py

```diff
@@ -0,0 +1,37 @@
+import os
+from pathlib import Path
+
+from paderbox.io.new_subdir import get_new_subdir
+
+
+def get_new_storage_dir(
+        experiment_name,
+        *,
+        id_naming='index',
+        mkdir=True,
+        prefix=None,
+        suffix=None,
+        consider_mpi=False,
+        dry_run=False,
+):
```

Comment:
Ads type annotations similar to the ones in paderbox?

Reply:
Done